package.json

```diff
@@ -13,15 +13,15 @@
     "test": "tap test.js"
   },
   "dependencies": {
-    "async-redis": "^1.1.7",
-    "redis-mock": "^0.49.0",
+    "async-redis": "^2.0.0",
```

Comment:
do you know if `async-redis` has any breaking changes that we should keep in mind?

Reply:
according to the [docs](https://www.npmjs.com/package/async-redis/v/2.0.0):

> This library does very little modification to the api of node_redis. It simply appends a promise resolving/rejecting callback for every command.

Reply:
i meant more about the major version change from `1.X.X` to `2.X.X`. i'd typically associate this with breaking changes or other large features that were introduced. i haven't found good documentation summarizing the update on the `async-redis` side yet though so trying to track that down.

Reply:
overall [this](https://github.com/moaxaca/async-redis/compare/v1.1.7...2.0.0) is looking ok to me. lots of security upgrades, some typescript definitions, github actions stuff, and some newer command support